src/main/java/io/pravega/connectors/flink/util/StreamId.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.connectors.flink.util;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Captures the fully qualified name of a stream. The convention to represent this as a
+ * single string is using [scope]/[stream].
+ */
+public class StreamId {
```

Comment:
I think what is best here is to commit this as is. If Pravega adopts this as part of the core code, we can then change this to use that class instead of having its own.